apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.YCSB/Google.Cloud.Spanner.Data.YCSB.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <StartupObject>Google.Cloud.Spanner.Data.YCSB.Program</StartupObject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Cloud.Spanner.Data" Version="1.0.0-beta08" />
```

Comment:
Any reason this isn't a project reference?

Reply:
i wanted to test against the public version, but a project reference is better going forward.